dirsrvtests/tests/suites/paged_results/paged_results_test.py

```diff
@@ -237,6 +237,7 @@ def paged_search(conn, suffix, controls, search_flt, searchreq_attrlist, abandon
     msgid = conn.search_ext(suffix, ldap.SCOPE_SUBTREE, search_flt, searchreq_attrlist, serverctrls=controls)
     log.info('Getting page %d' % (pages,))
     while True:
+        time.sleep(1)
```

Comment:
We have a stress test that runs pages search 1000 times:
https://github.com/389ds/389-ds-base/blob/a3d35219a31e97a5f1093e2f461cb0e3a67cdacc/dirsrvtests/tests/suites/paged_results/paged_results_test.py#L1258-L1268

Usually it takes about 300 seconds to complete, but with this change it takes significantly larger amount of time. I don't think we should do it here.

Reply:
I thought that the sleep time can be shorter. 
In my test, it worked even with `time.sleep(.1)` for rhel9.2 and rhel9.5. It may depend on the test environment, though.